packages/xo-web/src/xo-app/hub/recipes/recipe.js

```diff
@@ -51,10 +54,33 @@ export default decorate([
           size: 'medium',
         })
 
-        const { masterName, nbNodes, network, sr, sshKey } = recipeParams
+        const {
+          staticIpAddress,
+          masterIpAddress,
+          networkMask,
+          gatewayIpAddress,
+          masterName,
+          nbNodes,
+          network,
+          sr,
+          sshKey,
+        } = recipeParams
+
+        let workerNodeIps
+        if (staticIpAddress === true) {
+          workerNodeIps = {}
+          for (let i = 0; i < nbNodes; i++) {
+            const key = 'workerIpAddress' + (i + 1).toString()
+            workerNodeIps[key] = recipeParams[key]
+          }
+        }
 
         markRecipeAsCreating(RECIPE_INFO.id)
         const tag = await createKubernetesCluster({
+          masterIpAddress,
+          networkMask,
+          gatewayIpAddress,
+          workerNodeIps,
```

Comment:
Keep the properties sorted alphabetically to avoid unnecessary diffs in the future.